src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
```

Comment:
The interface name 'AreaCalculator' should not be a verb. A better name would be something like 'AreaComputable' or 'AreaProvider' to reflect that it provides the functionality to compute or provide an area.